lib/vrl/stdlib/src/parse_url.rs

```diff
@@ -92,6 +65,34 @@ impl Function for ParseUrl {
             default_known_ports,
         }))
     }
+
+    fn parameters(&self) -> &'static [Parameter] {
```

Comment:
This is exactly the same code as before, just rejigged by CLion to match the trait definition order.